plugins/plugin-chart-echarts/src/Bar/buildQuery.ts

```diff
@@ -0,0 +1,31 @@
+import { buildQueryContext, DrillDown, QueryFormData } from '@superset-ui/core';
+import { OwnState } from '../Pie/types';
+
+export default function buildQuery(formData: QueryFormData, options: any) {
+  const { drillDown } = formData;
+
+  return buildQueryContext(formData, baseQueryObject => {
+    let queryObject = {
+      ...baseQueryObject,
+    };
+
+    if (drillDown) {
+      const groupby = <string[]>formData.groupby;
+      const ownState = <OwnState>options.ownState || {
+        drilldown: DrillDown.fromHierarchy(groupby),
+      };
+      queryObject = {
+        ...queryObject,
+        ...(drillDown && {
+          groupby: [DrillDown.getColumn(ownState.drilldown, groupby)],
+          filters: [
+            ...(baseQueryObject.filters || []),
+            ...DrillDown.getFilters(ownState.drilldown, groupby),
+          ],
+        }),
+      };
+    }
+
+    return [queryObject];
+  });
+}
```

Comment:
is prettier not working?

Reply:
i had not run it because i had not heard of it til just now.  let me append another commit to this that prettifies all of my stuff to-date.

Reply:
@yevgenstylo: TIL prettier is the root cause of the trailing commas that you mentioned in https://github.com/askstylo/superset/pull/1#discussion_r712268442

Reply:
you can check the updated PR for yourself, but i can't see that prettier changed the lines you've pointed out.